src/SFA.DAS.ApprenticeFeedback.Domain.UnitTests/Entities/ApprenticeFeedbackTargetTests.cs

```diff
@@ -309,7 +311,29 @@ public void WhenCalling_ResetFeedbackTarget_SetsDefaultsOnTarget(Domain.Entities
         }
 
         [Test, RecursiveMoqAutoData]
-        public void WhenCalling_UpdateApprenticeshipFeedbackTarget_AndApprenticeshipIsPaused_SetsPausedDate(
+        public void WhenCalling_UpdateApprenticeshipFeedbackTarget_AndApprenticeshipIsStopped_AndLearnActEndDateIsSet_ButIsTransfer_SetInProgress_AndEndDateIsLearnActEndDate(
+            Learner learner,
+            ApplicationSettings settings,
+            Mock<IDateTimeHelper> dateTimeHelper,
+            Domain.Entities.ApprenticeFeedbackTarget target)
+        {
+            // Arrange
+            target.Withdrawn = false;
+            learner.CompletionStatus = 3;
+            learner.LearnActEndDate = DateTime.Now.AddDays(-7);
+            learner.IsTransfer = true;
+
+            // Act
+            target.UpdateApprenticeshipFeedbackTarget(learner, settings, dateTimeHelper.Object);
+
+            // Assert
+            target.EndDate.Should().Be(learner.LearnActEndDate);
+            target.Withdrawn.Should().BeFalse();
+            target.IsTransfer.Should().BeTrue();
```

Comment:
If one of these fails, subsequent assertions will be skipped. We can prevent this by wrapping the asserts inside an `AssertionScope`
```suggestion
            using (new AssertionScope())
            {
                 target.EndDate.Should().Be(learner.LearnActEndDate);
                 target.Withdrawn.Should().BeFalse();
                 target.IsTransfer.Should().BeTrue();
             }
```